tfsdk/serve_provider_test.go

```diff
@@ -431,6 +431,7 @@ func (t *testServeProvider) GetResources(_ context.Context) (map[string]Resource
 	return map[string]ResourceType{
 		"test_one":               testServeResourceTypeOne{},
 		"test_two":               testServeResourceTypeTwo{},
+		"test_three":             testServeResourceTypeThree{},
```

Comment:
Is it worth giving this a name like the validators below, to make it clear it's testing plan modifiers? 🤔